src/expr/src/expr/expr_binary_nonnull.rs

```diff
@@ -315,6 +350,54 @@ pub fn new_binary_expr(
                 },
             }
         }
+        // BitWise Operation
+        Type::PgBitwiseShiftLeft => {
+            gen_binary_expr_shift! {
+                gen_atm_impl,
+                l, r, ret,
+                general_shl,
+                {
+
+                },
+            }
+        }
+        Type::PgBitwiseShiftRight => {
+            gen_binary_expr_shift! {
+                gen_atm_impl,
+                l, r, ret,
+                general_shr,
+                {
+
+                },
+            }
+        }
+        Type::BitwiseAnd => {
+            gen_binary_expr_shift! {
```

Comment:
If this macro is used by both `shift` and `and` etc operations, maybe give a different name that works for both types.

Reply:
Ok I will change to gen_binary_expr_bitwise